MLendPoint/src/middleware/unzip.js

```diff
@@ -0,0 +1,17 @@
+const AdmZip = require("adm-zip");
+const path = require("path");
+
+class UnZip {
+    constructor() { }
+
+    extractZip(zipPath, outPath) {
```

Comment:
This function has keywords how to let, const and var. This last is not recommended for new uses